Quran/AyahInfoPersistenceStorage.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AyahInfoPersistenceStorage.swift
+//  Quran
+//
+//  Created by Ahmed El-Helw on 5/12/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+struct AyahInfoPersistenceStorage {
+
+    private let db: Connection
```

Comment:
should this be explicitly static, or is it ok to just have one, since we should only ever instantiate one (which will be part of the corresponding view controller?)